### ISSUE TITLE
Disable/remove "Notify about volume changes with keyboard" on Wayland

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt-panel#1934]
As title. "Show always notifications" is working normally.

##### Steps to Reproduce (for bugs)

1. Set the hook on "Notify about volume changes with keyboard"
2. Use the shortcuts
3. Nothing is shown

##### System Information

* LXQt Version: git
* Distribution & Version: arch, debian
* Compositors: labwc, wayfire

**Comment by stefonarch** [comment]
This probably depends as `lxqt-globalkeysd ` can't register keys under wayland yet.
Keyboard shortcuts are all set by compositors config files atm.

**Comment by stefonarch** [comment]
It would work using dbus: `qdbus org.kde.StatusNotifierWatcher /global_key_shortcuts/panel/volume2/mute org.lxqt.global_key_shortcuts.client.activated` for example but we have the [numbering issue](https://github.com/lxqt/lxqt-panel/issues/2126) - we cannot know if the panel has changed.

**Comment by tsujan** [comment]
I just cleaned up the issues list a little, for more important ones to become visible.  Changing its title and reopening ...

**Comment by stefonarch** [comment]
It's still to grey out under Wayland. 
It's possible to use external agents to show an OSD at volume changes by keyboard:

1) https://github.com/heyjuvi/avizo
* Add `avizo-service` to autostart
* Use "volumectl -|+|% ` in the `XF86Audio*` shortcuts in the compositor.

2) using `wob`, requires more fiddling but will work.

**Comment by isf63** [comment]
Does Wayland have a protocol for registering key bindings?

**Comment by stefonarch** [comment]
Not really, see the main discussion https://github.com/lxqt/lxqt-globalkeys/issues/282

**Comment by stefonarch** [comment]
Don't overestimate users... they see an option which doesn't work and will be frightened by such commands ;)

It's not about changing volume from keyboard which works out of the box on all our compositors except kwin_wayland (by using `wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle` but any command can be used including this qdbus ones). 
It' about this box which cannot be shown on Wayland when changing volume **by keyboard** by those types of shortcuts.

<img width="483" height="114" alt="Image" src="https://github.com/user-attachments/assets/49d51455-21c3-48fc-ab2b-9c1f7b0da655" />

**Comment by tsujan** [comment]
> It' about this box which cannot be shown on Wayland when changing volume by keyboard

That box is the tooltip. This is about showing *notifications*.

**Comment by tsujan** [comment]
I don't remember X11 — extra things can be done under it — but *that setting* is about notifications and works like a charm on Wayland.

**Comment by stefonarch** [comment]
> I don't remember X11 — extra things can be done under it — but _that setting_ is about notifications and works like a charm on Wayland.

"Notify volume changes with keyboard" does **not** work under Wayland, as setting. It works if you use that dbus command but that's independent of that setting. "Notify always" does.

**Comment by tsujan** [comment]
> It works if you use that dbus command but that's independent of that setting.

If I uncheck "Notify about volume changes with keyboard" and use the D-BUS command, no notification will be shown. Isn't it so for you? I'm on labwc.

**Comment by tsujan** [comment]
If you insist on guiding users, what about a tooltip that says "Effective on Wayland only with a D-BUS shortcut"? Advanced users will get it. But I think you'd agree that we shouldn't deprive (advanced or above average) users of something that ordinary users may not want to deal with.

**Comment by stefonarch** [comment]
To get the paths easier using `qdbus6 org.kde.StatusNotifierWatcher` then to navigate the GUI.

Anyway I was thinking of a way that works always (volume|volume2|volumeX):

```
#!/bin/sh
# actions: up|down|mute(=toggle mute)
action=$1
volume=$(qdbus6 org.kde.StatusNotifierWatcher | grep volume | head -n1)

qdbus6 org.kde.StatusNotifierWatcher $volume/$action org.lxqt.global_key_shortcuts.client.activated

```
Saving that as "volume" in $PATH it can be used in the Wayland settings for XF86AUDIO* commands (similar script for opening mainmenu or fancymenu can be made too).
